### DISCUSSION TITLE
Accessing "Historical Data"

### DISCUSSION THREAD

**Discussion by JcGKitten** [longhorn/longhorn#10021] [Q&A]
As explained in the documentation section about [volume size](https://longhorn.io/docs/archives/1.4.4/volumes-and-nodes/volume-size/) blocks deleted in the filesystem are only marked as unused. I'm wondering if it's possible to read this data again. I couldn't find anything about that.
Thx

**Comment by derekbit** [comment]
If you have snapshots, you can find the data.
